mapclientplugins/fieldworklowerlimb2sidegenerationstep/llstep.py

```diff
@@ -222,7 +222,7 @@ def _process(body, bodyLandmarks):
                     self.markerRadius, self.skinPad, *targetCoords
                 )
             except mocap_landmark_preprocess.InsufficientLandmarksError:
-                print('Insufficient landmarks for preprocessing {}'.format(body))
+                print(('Insufficient landmarks for preprocessing {}'.format(body)))
```

Comment:
Why does this line and others like it require the extra paren?

Reply:
I tried to use a script (2to3.py) for converting Python2 code to Python3 code and it did this for some reason. I tried to change most of them back but I missed some.